### PR TITLE
Refs #22478 - adjust execution_interface.js namespacing

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: lib/foreman_remote_execution/engine.rb [hunk 1]
```diff
@@ -120,7 +120,10 @@ class Engine < ::Rails::Engine
     # content twice.
     assets_to_precompile =
       Dir.chdir(root) do
-        Dir['app/assets/javascripts/**/*', 'app/assets/stylesheets/**/*'].map do |f|
+        Dir[
+          'app/assets/javascripts/foreman_remote_execution/**/*',
+          'app/assets/stylesheets/foreman_remote_execution**/*'
```
Comment (mmoll) [inline lib/foreman_remote_execution/engine.rb]: this was included by accident, will change ince the test run is finished, please don't merge

Comment (mmoll) [reply lib/foreman_remote_execution/engine.rb]: all fine now.

**Comment by ares** [comment]
Thanks @mmoll I'll do a new release this morning.

**Comment by iNecas** [comment]
Nica catch, thanks @mmoll!

**Pull request merged**